src/core/server/saved_objects/service/lib/search_dsl/query_params.ts

```diff
@@ -286,9 +286,9 @@ export function getQueryParams({
 
   if (ACLSearchParams) {
     const shouldClause: any = [];
-    if (ACLSearchParams.workspacePermissionModes && ACLSearchParams.principals) {
+    if (ACLSearchParams.permissionModes && ACLSearchParams.principals) {
       const workspacePermissionDSL = ACL.generateGetPermittedSavedObjectsQueryDSL(
```

Comment:
How about rename the `workspacePermissionDSL` to `permissionDSL`?

Reply:
Sure, good catch